content/en/docs/components/katib/overview.md

```diff
@@ -0,0 +1,205 @@
++++
+title = "Introduction to Katib"
+description = "Overview of Katib for hyperparameter tuning and neural architecture search"
+weight = 10
+                    
++++
+
+{{% beta-status
+  feedbacklink="https://github.com/kubeflow/katib/issues" %}}
+
+Katib is a Kubernetes-native project for automated machine learning (AutoML) —
+it's a system for hyperparameter tuning and neural architecture search.
+Katib supports a number of machine learning frameworks, including
+TensorFlow, MXNet, PyTorch, XGBoost, and others.
+Learn more about AutoML at [fast.ai](https://www.fast.ai/2018/07/16/auto-ml2/),
+[Google Cloud](https://cloud.google.com/automl),
+[Microsoft Azure](https://docs.microsoft.com/en-us/azure/machine-learning/concept-automated-ml#automl-in-azure-machine-learning) or
+[Amazon SageMaker](https://aws.amazon.com/blogs/aws/amazon-sagemaker-autopilot-fully-managed-automatic-machine-learning/).
+
+Katib supports a lot of various AutoML algorithms, such as
+Bayesian optimization, Tree of Parzen Estimators, Random Search,
+Covariance Matrix Adaptation Evolution Strategy, Hyperband,
+Neural Architecture Search based on ENAS, Differentiable Architecture Search and
+many more. Further algorithms are under development.
+
+The [Katib project](https://github.com/kubeflow/katib) is open source.
+The [developer guide](https://github.com/kubeflow/katib/blob/master/docs/developer-guide.md)
+is a good starting point for developers who want to contribute to the project.
+
+This page introduces the concepts of hyperparameter tuning, neural
```

Comment:
Maybe it makes sense to move this sentence towards the beginning, since every guide starts with: "Dear reader, this doc does XYZ. XYZ is a product that does ABC.."